test/sql/pg_supa.sql

```diff
@@ -0,0 +1,5 @@
+-- build extension
+create schema supa;
+create extension pg_supa with schema supa;
```

Comment:
Is it intentional to differentiate the sql extension(pg_supa) from the shared library(supautils)? If not, maybe we can change this to `create extension supautils` to maintain consistency.

Most extensions that I've seen use the same name for both the library and extension(e.g. you do don't do `create extension gis` for postgis).